src/styles.css

```
@@ -0,0 +1,2084 @@
+  --line-strong: rgb(242 239 231 / 28%);
+  --accent: #f48120;
+  --accent-hover: #ff963d;
+  --accent-ink: #171008;
+  --focus: #ffad66;
+  --header-bg: rgb(12 12 11 / 82%);
+  --shadow: 0 28px 90px rgb(0 0 0 / 34%);
+  --max-width: 1400px;
+  --display: "Barlow Condensed", "Arial Narrow", sans-serif;
+  --body: "Space Grotesk Variable", "Segoe UI", sans-serif;
+  --mono: "IBM Plex Mono", "Consolas", monospace;
+  font-family: var(--body);
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)